### ISSUE TITLE
Journal Entries appearing in wrong place

### ISSUE THREAD

**Issue by stebrough** [reapit/foundations#11323]
**Describe the bug**
As part of our automated property match system, we are posting journal entries to Reapit. These journal entries have always shown against the property as property match type entries. About a week ago, two customers reported they are now seeing these journal entries against the applicants as well. This is causing issues on the applicant journal due to the amount of entries.

**Expected behaviour**
The journal entries only show in the property, not the applicants. The intended behavior is the same as if the customer had done the match through Reapit itself, only showing journal entries on the property as a match.

**Additional context**
Sample webhook call emailed to sinfo@

**Comment by plittlewood-rpt** [comment]
Hi @stebrough it sounds like there was a problem previously if they weren't seeing the entries attached to the applicant. I have looked at the example payload you've sent and you are attaching the entry to both an applicant and a property, so the behaviour you've reported is expected. There has been some work over the last couple of weeks to migrate match data to a new database table which could be why this has suddenly started being visible in the system, however this is intended behaviour based on your examples so something wasn't working as it should have been prior. Hope this helps

**Comment by stebrough** [comment]
Thanks @plittlewood-rpt 

Are we doing this correctly then? We are aiming to mimic the current Reapit functionality exactly in terms of match journal entries. i.e. if a user did a match out in Reapit, would the match show in the applicant journal? If not, do we need a different AssociatedType?

**Comment by plittlewood-rpt** [comment]
Yes that looks correct. A match has two sides, the property and the applicant, and so it should be associated to both entities which from your example looks correct. Matches should be visible in the journals for both sides of the match.

**Comment by michaelwoodPSM** [comment]
@plittlewood-rpt couple more customers reporting issues to support desk, it's not clear where the issue might be at the moment whether API issue, product issue or partner side issue; 

https://reapit.atlassian.net/browse/RSD-296289 & https://reapit.atlassian.net/browse/RSD-295414 - matches in the journal created by II not correcting filtering out as matches in activity feed

https://reapit.atlassian.net/browse/RSD-295299 - matches not correctly recording as having been matched, so when users go to perform a match, already matched properties are appearing in results.

**Comment by plittlewood-rpt** [comment]
Hi @stebrough Michael has flagged this again with me and I've had another look. Part of our system which makes our APIs aware of customer product versions is not working correctly and the configuration for customers reporting this problem for some reason hasn't updated. As a result the API thinks they're on an older version of the software than they actually are. I've just updated the configuration for customers JOR and MPE. Do you know when you'll be writing more entries to either of those systems so I can confirm this as the problem whilst we look at a solution?

**Comment by stebrough** [comment]
Hi @plittlewood-rpt, I can't predict but I can keep an eye on it and let you know.

**Comment by plittlewood-rpt** [comment]
Hi @stebrough we've got to the bottom of this now and the immediate issue should be resolved. We'll need to move some data around in the respective customer's systems but we'll handle this separately.

**Comment by plittlewood-rpt** [comment]
Hi @stebrough  just to close the loop the underlying issue is indeed resolved and I've checked more recent data to make sure it is going into the correct table in the database. There's a few customers I just need to clean the data up for which will be done over the next day or so then that should complete this item.

**Comment by stebrough** [comment]
Thanks @plittlewood-rpt
